website/notifications/emails.py

```diff
@@ -44,6 +44,19 @@ def notify(event, user, node, timestamp, **context):
                 sent_users.extend(subscriptions[notification_type])
     return sent_users
 
+def notify_mentions(event, user, node, timestamp, **context):
+    event_type = utils.find_subscription_type(event)
+    sent_users = []
+    new_mentions = context.get('new_mentions', None)
+    for m in new_mentions:
```

Comment:
A `TypeError` will be raised here if `new_mentions` is assigned to `None` in the line above.